### PR TITLE
Publish erigonwatch ui for internal remote use

### PR DESCRIPTION
**Pull request by tosettil-polimi**
The following changes are backward compatible and do not affect the normal development and usage of the [erigontech/erigonwatch](https://github.com/erigontech/erigonwatch) repository.  
These modifications allow those who need it to publish the UI through a proxy so that it can be accessed remotely via a domain.  
The changes enable the UI to detect whether it is being served on a domain rather than on localhost (or 127.0.0.1); in such cases, requests made to the Erigon node are directed to the same domain with the `/api` prefix, to be managed through a proxy.

### CODE REVIEW

**Comment by mriccobene** [comment]
This PR is for the QA department to remotely monitor the machines running the tests and those producing the prebuilt databases. We are already using it in our infrastructure, publishing it with nginx and securing access with OAuth2.

**Comment by dvovk** [comment]
@mriccobene why don't you use standard remote connection flow ? Have you tried https://testdiag.erigon.io ?

File: src/Network/APIHandler.tsx [hunk 1]
```diff
@@ -166,14 +166,59 @@ export const heapProfileUrl = () => {
...
+				if (response.address) {
+					let address = response.address;
+					let isDefault = false;
+
+					if (address.startsWith("http")) {
+						address = address.substring(address.indexOf("//") + 2);
+						address = address.indexOf("/") >=0 ? address.substring(0, address.indexOf("/")) : address;
+					}
+
+					if (address === "127.0.0.1:6060" || address === "localhost:6060") isDefault = true;
```
Comment (dvovk) [inline src/Network/APIHandler.tsx:188]: This address may change as it setups by flag on diag cmd

**Comment by dvovk** [comment]
@mriccobene @tosettil-polimi please check https://github.com/erigontech/erigonwatch/pull/17 I did some updates and also include changes which I think is required to support proxy.  All what I did for it is added this two checks: 
`if (response.address.includes("127.0.0.1")) {
					//Check for proxy
					if (!window.location.origin.includes("127.0.0.1")) {
						beAddr = window.location.origin + "/api";
					}
				}`
Please check is it works as you expected or not

**Comment by mriccobene** [comment]
Superseded by #17. Can be closed.

**Pull request closed without merging**